test/apiv2/80-kube.at

```diff
@@ -37,13 +37,25 @@ t DELETE libpod/containers/$cid 200 .[0].Id=$cid
 
 # Make sure that kube-play works
 
-t POST   libpod/kube/play $YAML 200
-t DELETE libpod/kube/play $YAML 200
+t POST   libpod/kube/play $YAML 200 \
+  .Pods[0].ID~[0-9a-f]\\{64\\} \
+  .Pods[0].ContainerErrors=null \
+  .Pods[0].Containers[0]~[0-9a-f]\\{64\\}
+t DELETE libpod/kube/play $YAML 200 \
+  .Pods[0].ID~null \
+  .Pods[0].ContainerErrors=null \
+  .Pods[0].Containers[0]~null
 
 # Make sure that play-kube works
 
-t POST   libpod/play/kube $YAML 200
-t DELETE libpod/play/kube $YAML 200
+t POST   libpod/play/kube $YAML 200 \
+  .Pods[0].ID~[0-9a-f]\\{64\\} \
+  .Pods[0].ContainerErrors=null \
+  .Pods[0].Containers[0]~[0-9a-f]\\{64\\}
+t DELETE libpod/play/kube $YAML 200 \
+  .Pods[0].ID~null \
+  .Pods[0].ContainerErrors=null \
+  .Pods[0].Containers[0]~null
```

Comment:
OK, danger danger, this test is semi-reliably deadlocking my podman (#14929). @containers/podman-maintainers this looks very serious, please investigate ASAP.

Reply:
Also, can you check `.StopReport` and `.RmReport` here?